### PR TITLE
Add support for appsettings.ENVIRONMENT.json files

### PR DESCRIPTION
**Pull request by asser-dk**
I found it cumbersome manually having to add each of your `appsettings.ENVIRONMENT.json` config files for each of your projects.

With this commit files matching the pattern `appsettings.*.json` is also scanned.

I also attempted to clean up the autocomplete list by merging the values into a comma-separated string so the user can see the available options.

### CODE REVIEW

File: src/Resharper.ConfigurationSense/Extensions/ProjectExtensions.cs [hunk 1]
```diff
@@ -26,7 +27,7 @@ public static class ProjectExtensions
         {
             var configFiles = GetNetCoreJsonConfigFiles(project);
 
-            var settings = new HashSet<KeyValueSetting>(KeyValueSetting.KeyComparer);
+            var settings = new Dictionary<string, IList<string>>();
```
Comment (olsh) [inline src/Resharper.ConfigurationSense/Extensions/ProjectExtensions.cs]: Let's use linked list here
```suggestion
            var settings = new Dictionary<string, LinkedList<string>>();
```

File: src/Resharper.ConfigurationSense/Extensions/ProjectExtensions.cs [hunk 2]
```diff
@@ -81,11 +82,16 @@ public static class ProjectExtensions
                         }
                     }
 
-                    settings.Add(new KeyValueSetting(formattedPath, property.Value.ToString()));
+                    if (!settings.ContainsKey(formattedPath))
```
Comment (olsh) [inline src/Resharper.ConfigurationSense/Extensions/ProjectExtensions.cs]: We can optimize this to access the dictionary only once with the `TryGet` method

**Review by olsh — CHANGES_REQUESTED** [review]
Thank you for the PR! 👍

**Comment by asser-dk** [comment]
replaced the list and the dictionary access

**Comment by olsh** [comment]
Thank you for the contribution 👍 
Merged in #10

**Pull request closed without merging**